tests/conftest.py

```diff
@@ -36,3 +39,24 @@ def spark_database(containers):
 def cleanup_register():
     yield
     cohort_registry.reset()
+
+
+@pytest.fixture
+def OldCohortWithPopulation():
+    class OldCohort:
+        def __init_subclass__(cls):
+            if not hasattr(cls, "population"):
+                cls.population = table("practice_registrations").exists()
+
+    yield OldCohort
```

Comment:
How come this needs to be a fixture? Couldn't you get the same effect by just importing this class into each of the test modules and then you wouldn't need to include it in every single test function signature.

Reply:
It could; in fact I did it like that before and can't remember why I changed it.

Reply:
I think it may have just been that I was doing the new version as a fixture; I'll change it back